src/components/containers/SSIButtonsContainer/index.tsx

```diff
@@ -52,7 +52,7 @@ class SSIButtonsContainer extends PureComponent<Props, IState> {
             onPress={secondaryButton.onPress}
             disabled={secondaryButton.disabled}
             // TODO move styling to styled components (currently there is an issue where this styling prop is not being set correctly)
-            style={{height: 42, minWidth: 145, width: primaryButton ? undefined : 300}}
+            style={{height: 42, minWidth: 130, width: primaryButton ? undefined : 300}}
```

Comment:
We have to change the min width. Min width should depend on the smallest size screen . Having a larger value. Like 132 will make the buttons cropped from the sides. Hence. This value is correct. I have seen some other screens on which the impact is not noticable.
If required I can add styling to those areas instead of reverting this. If required we can see each screen which have these buttons. I hope the result is acceptable.